build/gulpfile.mjs

```diff
@@ -37,7 +39,17 @@ gulp.task(transpileClientTask);
 const compileClientTask = task.define('compile-client', task.series(util.rimraf('out'), compileApiProposalNamesTask, compileTask('src', 'out', false)));
 gulp.task(compileClientTask);
 
-const watchClientTask = task.define('watch-client', task.series(util.rimraf('out'), task.parallel(watchTask('out', false), watchApiProposalNamesTask)));
+const watchClientEsbuildTask = task.define('watch-client-esbuild', watchTask('out', false, 'src', { useEsbuild: true, logTopic: 'src-transpile', skipInitial: true }));
+const watchClientCheckTask = task.define('watch-client-check', watchTask('out', false, 'src', { useEsbuild: false, noEmit: true, logTopic: 'src-check' }));
+const watchClientTask = task.define('watch-client', task.series(
+	util.rimraf('out'),
+	task.parallel(transpileClientSWCTask, compileApiProposalNamesTask),
+	task.define('client-ready', () => {
+		fancyLog(ansiColors.bgGreen.black('*** Finished transpiling the client. Ready to launch for debugging! *** '));
+		return Promise.resolve();
+	}),
+	task.parallel(watchClientEsbuildTask, watchClientCheckTask, watchApiProposalNamesTask)
```

Comment:
Why do we need first `parallel(transpile, compile)` and why isn't it just `parallel(watch, watch, watch)`? Shouldn't the first watch-transpile do all files and be fast from then one?